draft-ietf-quic-applicability.md

```diff
@@ -709,6 +709,22 @@ new version uses the same process in reverse.  Servers disable validation of the
 old version, stop sending the old version in Version Negotiation packets, then
 the old version is no longer accepted.
 
+# Unreliable Datagram Service over QUIC
+
+{{?I-D.draft-ietf-quic-datagram}} specifies an QUIC extension to enable sending
+and receiving unreliable datagrams over QUIC. Other than with the use of UDP
+only, the datagram service in QUIC is congestion-controlled, and thereby
+supporting application designers to address the requirements in {{?RFC8085}}.
+QUIC datagrams are not flow-controlled and data chunks may get drop if the
+receiver is overloaded.
```

Comment:
I'd move this last sentence into the next paragraph, because lack of flow control is just another way in which Datagrams are not reliable.